Formula/nsq.rb

```diff
@@ -1,29 +1,21 @@
 class Nsq < Formula
   desc "Realtime distributed messaging platform"
   homepage "https://nsq.io/"
-  url "https://github.com/nsqio/nsq/archive/v0.3.8.tar.gz"
-  sha256 "d9107cdfe218523a74ee801caaa97968becb4b82dae7085dbb52d05c25028ff3"
+  url "https://github.com/nsqio/nsq/releases/download/v0.2.31/nsq-0.2.31.darwin-amd64.go1.3.1.tar.gz"
+  sha256 "ffc40ac7e7bf70de08bf8783aeb662b3aa974b16f4417f21a0fc16d5582724e8"
```

Comment:
This checksum doesn't match for me. Here's one that does match if you can confirm: `0e17cce422287d58a65e0fea0142381c9e6cd8cb6033f8aae52256dcb7575c8d`

You can check it by doing `echo "<checksum here> <filename>" | sha256sum --check`

Reply:
The `0e17` checksum fails for me.

Reply:
I was downloading this file with curl and it was getting redirected, so my checksum was from the HTML. This checksum `ffc40` matches mine locally now, going to approve.